### ISSUE TITLE
play.tv3.lv: detection

### ISSUE THREAD

**Issue by mozilla343** [uBlockOrigin/uAssets#22407]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://play.tv3.lv/`

### Category

detection

### Description

This site might have geoblock set in place which could prevent videos from loading, so while you test it, please use Latvia (LV) VPN if necessary.

Please see details in screenshot, when this filter is active then videos don't play. 
![image](https://github.com/uBlockOrigin/uAssets/assets/62311944/f4327df9-40a8-4798-92d0-e2e3b742381e)

In the red area it says "error occurred, unexpected error has occurred in TV3Play. Try again"
After I allow this AdManager.js script file in ublock origin videos play again normally. 

AdManager.js is coming from mssl.fwmrm.net subdomain, but while you are making filter fix updates, this subdomain: 5d423.v.fwmrm.net must still be blocked, as this is the source where actual video ads are coming from.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/62311944/f4327df9-40a8-4798-92d0-e2e3b742381e)

![image](https://github.com/uBlockOrigin/uAssets/assets/62311944/90a7d03c-68c3-4329-88ac-dff5b41a196c)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.55.0
Chromium: 121
filterset (summary):
 network: 140708
 cosmetic: 67424
 scriptlet: 21875
 html: 0
listset (total-discarded, last-updated):
 added:
  EST-0: 464-1, now
  LTU-0: 1252-2, now
  LVA-0: 396-4, now
  RUS-0: 39946-17, now
 default:
  user-filters: 4-0, never
  easylist: 80374-71, now
  easyprivacy: 50142-147, now
  plowe-0: 3787-1189, now
  ublock-badware: 7849-140, now
  ublock-filters: 37157-235, now
  ublock-privacy: 998-7, now
  ublock-quick-fixes: 199-4, now
  ublock-unbreak: 2206-34, now
  urlhaus-1: 7240-0, now
filterset (user): [array of 4 redacted]
trustedset:
 added: [array of 5 redacted]
userSettings:
 advancedUserEnabled: true
 cloudStorageEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 684 ms (selfie)
 maxAssetCacheWait: 368 ms
 cacheBackend: browser.storage.local
popupPanel:
 blocked: 13
 network:
  tv3.lv: 1
  chartbeat.com: 1
  fwmrm.net: 1
  gemius.pl: 2
  go3.tv: 7
  googletagmanager.com: 1
 extended:
  ##.is-visible.c-paused-overlay
  ##+js(set-constant, rwt, noopFunc)
```
</details>

**Comment by dimisa-RUAdList** [comment]
@mozilla343 
Check: `play.tv3.lt,play.tv3.lv,tv3play.skaties.lv##+js(set, Object.prototype.isNoAds, {})
play.tv3.lt,play.tv3.lv,tv3play.skaties.lv#@##ad-lead
play.tv3.lt,play.tv3.lv,tv3play.skaties.lv#@##AD_Top
play.tv3.lt,play.tv3.lv,tv3play.skaties.lv#@##AdHeader
play.tv3.lt,play.tv3.lv,tv3play.skaties.lv#@##homead`

**Comment by dimisa-RUAdList** [comment]
@ghajini 
Indeed, when using uBlock filters, these exceptions are not needed, since the ghide for these domains is contained there. I just copied what was previously used in the RU AdList filter for Adblock Plus.

**Comment by mozilla343** [comment]
> add `play.tv3.lt,play.tv3.lv,tv3play.skaties.lv##+js(set, Object.prototype.isNoAds, {})` in my filters in ublock settings

I did that and then it works yes, but why it didn't come automatically with filter (manually triggered) updates ? Because my relatives have the same problem and it's not conveniently to fix it manually as they live far away.
